packages/web/docs/src/app/gateway/gateway-feature-tabs.tsx

```
@@ -0,0 +1,100 @@
+      title: 'CORS and CSRF Prevention',
+      description:
+        'Shields against common web security vulnerabilities, securing your applications from unauthorized inter-domain interactions.',
+    },
+  ],
+};
+
+export function GatewayFeatureTabs(props: { className?: string }) {
+  return (
+    <FeatureTabs
+      highlights={highlights}
+      icons={[<PerformanceIcon />, <SecurityIcon />]}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)